keyboards/handwired/polly40/readme.md

```diff
@@ -0,0 +1,17 @@
+# Polly40
+
+A handwired keyboard on a Pro micro Atmega32u4 controller
+
+![ Polly ](https://i.imgur.com/QJCQcFAh.jpg)
+
+
+* Keyboard Maintainer: Paul Enrico N. Viola (https://github.com/PollyV1)
+* Hardware Supported: Pro micro Atmega32u4 controller
+* Hardware Availability: https://github.com/PollyV1/Polly40/
+
+
+Make example for this keyboard (after setting up your build environment):
+
+    make polly/default:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
This file's contents need to conform to the [template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md?plain=1).